src/app/components/Icons/index.ts

```diff
@@ -0,0 +1,6 @@
+import SettingsIcon from './SettingsIcon';
+import InfoIcon from './InfoIcon';
+import HomeIcon from './HomeIcon';
+import FavoritesIcon from './FavoritesIcon';
+
+export { SettingsIcon, InfoIcon, HomeIcon, FavoritesIcon };
```

Comment:
In general, I like this approach.
Keep in mind that this will import and bundle all components, even if you only need some of them. 
That's why [Tree Shaking](https://webpack.js.org/guides/tree-shaking/) (or [Tree Shaking in rollup](https://www.rollupjs.org/guide/en/#tree-shaking)) was invented.

Reply:
👍